gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -281,6 +282,7 @@ static void addJobStatusToStateStore(org.apache.gobblin.configuration.State jobS
       stateStore.put(storeName, tableName, jobStatus);
       if (isNewStateTransitionToFinal(jobStatus, states)) {
         eventProducer.emitObservabilityEvent(jobStatus);
+        eventBus.post(new JobStatusEvent(jobStatus));
```

Comment:
why is this change here? can u add a comment to explain who the listener is here of the job status event?